### DISCUSSION TITLE
Full clickable method

### DISCUSSION THREAD

**Discussion by userjlegwork** [dlcastillop/paylink#17] [Ideas]
What about making the payment methods clickable (when you hover over them feels like a button, but when clicked nothing happens)

**Comment by dlcastillop** [comment]
Each payment method has two buttons: for copy and the QR code. I think it will be unnecessary that feature.

**Reply by userjlegwork** [reply]
It has two options but one is the main one, that's my perspective, I'll do the change on my fork 👍
